meta/revierstil/logo.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?> <!-- Created with Vectornator for iOS (http://vectornator.io/) --><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg height="100%" style="fill-rule:nonzero;clip-rule:evenodd;stroke-linecap:round;stroke-linejoin:round;" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="100%" xmlns:vectornator="http://vectornator.io" version="1.1" viewBox="0 0 121.3 136.3">
+<metadata>
+<vectornator:setting key="DimensionsVisible" value="1"/>
+<vectornator:setting key="PencilOnly" value="0"/>
+<vectornator:setting key="SnapToPoints" value="0"/>
+<vectornator:setting key="OutlineMode" value="0"/>
+<vectornator:setting key="CMYKEnabledKey" value="0"/>
+<vectornator:setting key="RulersVisible" value="1"/>
+<vectornator:setting key="SnapToEdges" value="0"/>
+<vectornator:setting key="GuidesVisible" value="1"/>
+<vectornator:setting key="DisplayWhiteBackground" value="0"/>
+<vectornator:setting key="doHistoryDisabled" value="0"/>
+<vectornator:setting key="SnapToGuides" value="1"/>
+<vectornator:setting key="TimeLapseWatermarkDisabled" value="0"/>
+<vectornator:setting key="Units" value="Points"/>
+<vectornator:setting key="DynamicGuides" value="0"/>
+<vectornator:setting key="IsolateActiveLayer" value="0"/>
+<vectornator:setting key="SnapToGrid" value="0"/>
+</metadata>
+<defs/>
+<g id="Untitled" vectornator:layerName="Untitled">
+<g opacity="1">
+<path d="M-1+66.9L0+67.7C0+67.3+0+66.8+0+66.4L-1+66.9Z" opacity="1" fill="none"/>
+<path d="M98.2+17.3C94.5+12.4+89.8+5.4+76.3+2C71.4+0.8+65.4+0+57.9+0C54.4+0+50.1+0+45.3+0C37.6+0+28.5+0+19.2+0C12.3+0+7.9+3.5+5.1+8L45.4+45.4L119.9+45.4C119.8+45.3+103.2+23.9+98.2+17.3Z" opacity="1" fill="#d3de2a"/>
+<path d="M5.1+8C2.9+11.5+1.7+15.6+1+19.1C0.3+22.5+0+25.9+0+29.3C0+31.9+0+37.1+0+43.9C0+50.3+0+58.1+0+66.5C0+66.9+0+67.4+0+67.8C0+98.8+0+136.3+0+136.3C0+136.3+21.4+119.7+28.1+114.7C31.6+112+36.3+108.8+39.9+102.1C43.1+96.2+45.4+87.7+45.4+74.4C45.4+67.1+45.4+56.8+45.4+45.4L5.1+8Z" opacity="1" fill="#606161"/>
+</g>
+</g>
+</svg>
```

Comment:
Please minify the SVG as described in the README :)

Reply:
The SVG is minified now :-)